pytrip/vdx.py

```diff
@@ -1100,7 +1102,14 @@ def create_dicom_contours(self):
             # if CT cube is loaded we extract DICOM representation of the cube (1 dicom per slice)
             # and select DICOM object for current slice based on slice position
             # it is time consuming as for each call of this method we generate full DICOM representation (improve!)
-            candidates = [dcm for dcm in self.cube.create_dicom() if dcm.SliceLocation == self.get_position()]
+
+            from pytrip.util import float_compare
+
+            _eps = 0.00001
+            
+            candidates = [dcm for dcm in self.cube.create_dicom() if float_compare(dcm.SliceLocation,
```

Comment:
You could use this one https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.isclose.html instead of writing method yourself.